test/integration/lib/foxbox_process_manager.js

```diff
@@ -10,7 +10,7 @@ var config = new Config('./test/integration/lib/config/foxbox.js');
 var FOXBOX_STARTUP_WAIT_TIME_IN_MS = 5000;
 var foxboxInstance;
 
-var helper = (function() {
+var foxbox_process_manager = (function() {
```

Comment:
Yeah, let's update all the case issues in a follow up that will enforce that with jshint